Project.toml

```diff
@@ -33,18 +40,19 @@ OrdinaryDiffEq = "6"
 PDEBase = "0.1.4"
 PrecompileTools = "1"
 SafeTestsets = "0.0.1"
-SciMLBase = "1.90, 1.91"
+SciMLBase = "1.94"
 StaticArrays = "1"
 SymbolicUtils = "1"
 Symbolics = "5"
 TermInterface = "0.2, 0.3"
 julia = "1.6"
 
 [extras]
+JuliaSimCompiler = "8391cb6b-4921-5777-4e45-fd9aab8cb88d"
```

Comment:
This would require having access to the `JuliaSimCompiler` at all times to run the tests, making it synonymous with a direct dependency (and less of an extension) in the context of running tests. We should be explicitly `Pkg.add`-ing `JuliaSimCompiler` instead in the relevant test matrices.
```suggestion
```